CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [7.3.1]
+
+### Added
+* Turn off hyp3 ACCESS spend by zeroing the max VCPUs in the associated deployment.
+* Reduce product lifetime in hyp3 ACCESS deployment to 14 days.
```

Comment:
@cmarshak code says 30 days and changelog says 14; which did you intend?

Reply:
14 - Thanks! Let's release tomorrow because we are still processing :)